src/main/java/com/reckue/post/services/realizations/CommentServiceRealization.java

```diff
@@ -142,7 +136,7 @@ public Comment update(Comment comment) {
             case "postId":
                 return findAllAndSortByPostId();
             case "published":
```

Comment:
May be you need to change "published" to "created", and then change it in line 131.